### PR TITLE
Automatic TGS DMAPI Update

### PR DESCRIPTION
**Pull request by Cyberboss**
This pull request updates the TGS DMAPI to the latest version. Please note any breaking or unimplemented changes before merging.

### CODE REVIEW

File: code/modules/tgs/v3210/api.dm [hunk 1]
```diff
@@ -92,14 +92,14 @@
...
-	call(SERVICE_INTERFACE_DLL, SERVICE_INTERFACE_FUNCTION)(instance_name, command) //trust no retval
+	call(SERVICE_INTERFACE_DLL, SERVICE_INTERFACE_FUNCTION)(instance_name, command)	//trust no retval
 	return TRUE
 
 /datum/tgs_api/v3210/OnTopic(T)
 	var/list/params = params2list(T)
 	var/their_sCK = params[SERVICE_CMD_PARAM_KEY]
 	if(!their_sCK)
-		return FALSE //continue world/Topic
+		return FALSE	//continue world/Topic
```
Comment (Mothblocks) [inline code/modules/tgs/v3210/api.dm:102]: @Cyberboss why is it doing alignment indenting

Comment (Mothblocks) [reply code/modules/tgs/v3210/api.dm:102]: thats like this whole pr, can you fix tgs to not have that?

**Comment by Mothblocks** [comment]
cc @Cyberboss -- what's with the alignment indenting? can you fix that in tgs dmapi?

**Comment by Cyberboss** [comment]
Should update soonish

**Comment by Mothblocks** [comment]
Double-check that this diff is correct, is our only change buffing the #define?

**Comment by Cyberboss** [comment]
Should update soonish

> 
> 
> Double-check that this diff is correct, is our only change buffing the #define?
> 
> Or is it because you needed to update DMAPI to fit the standards, and this is just a side effect of that update (since we already had it)?

The latter

**Pull request merged**